### ISSUE TITLE
Double event 'after-hide'

### ISSUE THREAD

**Issue by UncleVic** [maxogden/menubar#185]
I think should  correct like this **|| !menubar.window.isVisible()**
 
```
    function hideWindow() {
      if (supportsTrayHighlightState) menubar.tray.setHighlightMode('never')
      if (!menubar.window || !menubar.window.isVisible()) return
      menubar.emit('hide')
      menubar.window.hide()
      menubar.emit('after-hide')
    }

```

**Comment by amaurymartiny** [comment]
Note: the double event only happens when we click on the icon again to hide.

@UncleVic Your solution works. Would you like to submit a PR?

**Comment by amaurymartiny** [comment]
Released in [`menubar@6.0.4`](https://www.npmjs.com/package/menubar/v/6.0.4) 🚀
